docs/docs/known-issues.md

```diff
@@ -1,33 +1,30 @@
-# Known issues & limitations
+## **Known issues and limitations**
 
-Please be aware of the following known issues and limitations before running `hydra-node` on the Cardano `--mainnet`; as an incredibly technical project, Hydra in its current form requires an elevated level of understanding of the underlying infrastructure. Without the expertise required to operate a `hydra-node`, users may put their funds at risk if they are unfamiliar with the implementation and usage process.
+Before running a Hydra node on the Cardano mainnet, it is important to be aware of several known issues and limitations. Operating a Hydra node requires a deep understanding of the underlying infrastructure, and you may risk your funds if you are unfamiliar with the implementation and usage processes.
 
-### Head protocol limits
+**Head protocol limits**
 
-As transaction sizes and execution budgets are limited on Cardano:
+Due to the limitations on transaction sizes and execution budgets on Cardano, the Hydra protocol has the following constraints:
 
-- The protocol can only handle a maximum number of participants in a Head (see [cost of collectcom transaction](/benchmarks/transaction-cost/#cost-of-collectcom-transaction)). Upon startup, the `hydra-node` will inform you of the current configured maximum when trying to configure too many peers.
+**Maximum number of participants. **The protocol can handle only a limited number of participants in a Hydra head. The exact limit is determined by the cost of the ``collectcom`` transaction. When attempting to configure too many peers, the Hydra node will inform you of the current maximum allowed participants upon startup. For more information,[ see the cost of the collection transaction.](https://hydra.family/head-protocol/unstable/benchmarks/transaction-cost/#cost-of-collectcom-transaction)
 
-It's currently possible to be denied access to funds by other protocol participants at various stages in a Hydra Head because of the complexity or size of the UTxO being committed or created while the Head is open:
+* **Access to funds:**
+    * Participants may be denied access to their funds by other protocol participants at different stages within a Hydra head because of the complexity or size of the UTXO being committed or created while the head is open.
```

Comment:
I'd suggest to make this the intro sentence of this paragraph.

```suggestion
* **Access to funds.** Participants may be denied access to their funds by other protocol participants at different stages within a Hydra head because of the complexity or size of the UTXO being committed or created while the head is open:
```

Reply:
Updated